src/ts/audiotrackutils.ts

```diff
@@ -46,7 +47,7 @@ export class AudioTrackSwitchHandler {
   private addAudioTrack = (event: AudioTrackEvent) => {
     const audioTrack = event.track;
     if (!this.listElement.hasItem(audioTrack.id)) {
-      this.listElement.addItem(audioTrack.id, audioTrack.label);
+      this.listElement.addItem(audioTrack.id, getLocalizedText(i18n.t(audioTrack.label)));
```

Comment:
This is unnecessarily complicated, why two function calls? This should simply be `i18n.something(audioTrack.label)` instead of `getLocalizedText(i18n.t(audioTrack.label))` (applies in other places too).